plugins/modules/net_tools/pritunl/pritunl_user.py

```diff
@@ -0,0 +1,360 @@
+                            "user_id": users[0]["id"],
+                            "user_data": user_params,
+                        },
+                    )
+                )
+
+                result["changed"] = True
+                result["response"] = response
+
+            except PritunlException:
+                module.fail_json(
+                    msg="Could not update Pritunl user %s from %s organization"
+                    % (user_name, org_name)
+                )
+        else:
+            result["changed"] = False
+            result["response"] = users
+    else:
+        try:
+            response = post_pritunl_user(
+                **dict_merge(
+                    get_pritunl_settings(module),
+                    {
+                        "organization_id": org_id,
+                        "user_data": user_params,
+                    },
+                )
+            )
+            result["changed"] = True
+            result["response"] = response
+        except PritunlException:
+            module.fail_json(
+                msg="Could not add Pritunl user %s to %s organization"
+                % (user_params["name"], org_name)
+            )
+
+    module.exit_json(**result)
+
+
+def remove_pritunl_user(module):
+    result = {}
+
+    org_name = module.params.get("organization")
+    user_name = module.params.get("user_name")
+
+    org_obj_list = []
+    try:
+        org_obj_list = list_pritunl_organizations(
+            **dict_merge(
+                get_pritunl_settings(module),
+                {
+                    "filters": {"name": org_name},
+                },
+            )
+        )
+    except PritunlException:
+        pass
+
+    if len(org_obj_list) == 0:
+        module.fail_json(
+            msg="Can not remove user '%s' from a non existing organization '%s'"
+            % (user_name, org_name)
+        )
+
+    org_id = org_obj_list[0]["id"]
+
+    # Grab existing users from this org
+    users = list_pritunl_users(
+        **dict_merge(
+            get_pritunl_settings(module),
+            {
+                "organization_id": org_id,
+                "filters": {"name": user_name},
+            },
+        )
+    )
+
+    # Check if the pritunl user exists, if not, do nothing
+    if len(users) == 0:
+        result["changed"] = False
+        result["response"] = {}
+
+    # Otherwise remove the org from Pritunl
+    else:
+        try:
+            response = delete_pritunl_user(
+                **dict_merge(
+                    get_pritunl_settings(module),
+                    {
+                        "organization_id": org_id,
+                        "user_id": users[0]["id"],
+                    },
+                )
+            )
+            result["changed"] = True
+            result["response"] = response
+        except PritunlException:
+            module.fail_json(
+                msg="Could not remove user %s from organization %s from Pritunl"
```

Comment:
Suggestion: instead of putting `try/except` inside each function, raise the exception with the message that should be in the `fail_json()` call, as in:
```python
    raise PritunlException("Could not remove user...")
```
then in `main()` you put the `try/except` around the whole thing, as in:
```python
    try:
        if state == "present":
            add_or_update_pritunl_user(module)
        elif state == "absent":
            remove_pritunl_user(module)
    except PritunlException as e:
        module.fail_json(msg=str(e))
```